pandas/tests/arrays/test_integer.py

```diff
@@ -339,6 +339,19 @@ def test_error(self, data, all_arithmetic_operators):
         with pytest.raises(NotImplementedError):
             opa(np.arange(len(s)).reshape(-1, len(s)))
 
+    @pytest.mark.parametrize("zero, negative", [(0, False), (0.0, False), (-0.0, True)])
```

Comment:
might be good to make a fixture for zeros as we do a lot of these types of tests for series ops, cc @jbrockmendel